es/_posts/2023-08-03-049.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+pretitle:
+title: Fusionando
+subtitle: ramas
+command: git merge origen destino
+descriptors:
+  - command: comando para juntar ramas
+  - part1: rama con los cambios a
+    aplicar
+  - part2: rama que debe recibir
+    los cambios
+info: usa `git merge source target` para fusionar la rama de origen en la rama destino
+number: "049"
+author: "@jtemporal"
+mermaid: true
+use_static_image: true
```

Comment:
Since you are using this, can you generate the SVG with the command below and add the image? Another option is to remove this line. Thanks :)

```
python3 scripts/generate_images_only.py "049.md"
```